backend/azure-functions/src/IsSupervisor/index.ts

```diff
@@ -0,0 +1,27 @@
+import { PassedAuthenticationResult, authenticate } from '../Auth/facebook-auth';
+import { Context, HttpRequest } from 'azure-functions-ts-essentials';
+
+const allowedUserIds: string[] = [
+  // TODO: add users
+];
+
+const IsSupervisor = async function (context: Context, req: HttpRequest): Promise<void> {
+  const authenticatedUser: PassedAuthenticationResult = await authenticate(req, context, true) as PassedAuthenticationResult;
+  if (!(authenticatedUser instanceof PassedAuthenticationResult)) return;
+
+  if (allowedUserIds.indexOf(authenticatedUser.userId) > -1) {
+    context.res = {
+      status: 200,
+      body: true,
+    };
+  } else {
+    context.res = {
+      status: 401,
```

Comment:
Makes more sense to return here 200, and to return the result in a body json. It's not an unauthorized operation to ask if I'm authorized.